src/OrchardCore.Modules/OrchardCore.Taxonomies/AdminMenu.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Navigation;
+using OrchardCore.Taxonomies.Settings;
+
+namespace OrchardCore.Taxonomies
+{
+    public class AdminMenu : INavigationProvider
+    {
+        private readonly IStringLocalizer S;
+
+        public AdminMenu(IStringLocalizer<AdminMenu> localizer)
+        {
+            S = localizer;
+        }
+
+        public Task BuildNavigationAsync(string name, NavigationBuilder builder)
+        {
+            if (!String.Equals(name, "admin", StringComparison.OrdinalIgnoreCase))
+            {
+                return Task.CompletedTask;
+            }
+
+            builder.Add(S["Configuration"], configuration => configuration
+                   .Add(S["Settings"], "1", settings => settings
+                        .Add(S["Taxonomy Contents List Filters"], S["Taxonomy Contents List Filters"].PrefixPosition(), admt => admt
```

Comment:
This title is quite long `Taxonomy Contents List Filters`. 
You can't even magine how many words it takes to translate it in French ;-)

Reply:
Changed to `Taxonomy Filters` :)